configure.ac

```diff
@@ -1869,6 +1869,52 @@
         fi
     fi
 
+#libevent if hiredis is enabled
+    AC_ARG_ENABLE(libevent,
+            AS_HELP_STRING([--enable-libevent],[Enable libevent for async calls for redis ]),
```

Comment:
I don't like the idea of having a new flag in configure. Could we have it at least in auto mode so if hiredis if selected then we try to get libevent ? and fallback to no libevent if not available.